animation_nodes/nodes/falloff/point_distance_falloff.pyx

```diff
@@ -1,19 +1,56 @@
 import bpy
+from bpy.props import *
 from ... data_structures cimport BaseFalloff
-from ... base_types import AnimationNode
 from ... math cimport Vector3, setVector3, distanceVec3
+from ... base_types import AnimationNode, VectorizedSocket
+
+mixListTypeItems = [
+    ("MAX", "Max", "", "NONE", 0),
+    ("ADD", "Add", "", "NONE", 1),
+]
 
 class PointDistanceFalloffNode(bpy.types.Node, AnimationNode):
     bl_idname = "an_PointDistanceFalloffNode"
     bl_label = "Point Distance Falloff"
 
+    __annotations__ = {}
+
+    __annotations__["mixListType"] = EnumProperty(name = "Mix List Type", default = "MAX",
+        items = mixListTypeItems, update = AnimationNode.refresh)
```

Comment:
Is `refresh` really necessary here?

Reply:
@OmarEmaraDev when i used ``propertyChanged`` it dosen't update

Reply:
@harisreedhar Update what?

Reply:
from changing ``Max`` to ``Add`` it dosen't work

Reply:
@harisreedhar Ah. You are embedding/expanding the value of the enum in the execution code `'{self.mixListType}'`, which means the execution code changes and have to be recompiled every time the enum change. In cases where the execution code change, you can use `executionCodeChanged` instead of `propertyChanged`. But you can just write the code such that it doesn't change by simply writing `self.mixListType` without expanding it. But since I suggested we use `getExecutionFunctionName`, this won't be necessary either.

Reply:
@OmarEmaraDev  Where should i look to learn how AN execution works?